MartinP460/react-denmark-map
# Provide explicit type values for the fields in the different area types

### Proposed changes

Casted area data arrays to const and created a generic type for area objects from that.

### How to test

1. Go to any Storybook component and verify that the type of each of the fields in an area object has a union of the available data.
